lib/authorizations.js

```diff
@@ -0,0 +1,57 @@
+/*!
+ * Copyright (c) 2019-2023 Digital Bazaar, Inc. All rights reserved.
+ */
+import * as bedrock from '@bedrock/core';
+import {httpClient} from '@digitalbazaar/http-client';
```

Comment:
We're also going to need `@bedrock/https-agent` here and will need to pass its exported `agent` to http-client in its options below to handle dev-mode SSL cert stuff.

Unfortunately, the tests don't seem to cover this, I presume because of mocked API endpoints or some such -- but as soon as this would be used in a dev environment I would expect it to fail. Of course, maybe we are never supposed to use turnstile in a dev environment? I suggest we add this agent stuff anyway to ensure we're following our usual pattern to help reduce any unforeseen dev maintenance headaches.